bitbucket-pipelines.yml

```diff
@@ -24,7 +24,6 @@ pipelines:
             libignition-math6-dev
             libignition-msgs5-dev
             libignition-tools-dev
-            libignition-plugin-dev
```

Comment:
do we need this file?

Reply:
I think so, see https://github.com/ignition-tooling/release-tools/issues/203